src/service/app.py

```diff
@@ -3,8 +3,8 @@
 '''
 import os
 
-from common.git_commit import GIT_COMMIT
-from common.version import VERSION
+from src.common.git_commit import GIT_COMMIT
+from src.common.version import VERSION
```

Comment:
I'm not sure whether the common python practice is to include src in the import path or not. If not, we'd need to set the PYTHONPATH to /src. 

Personally I don't really care either way as long as we're consistent.